readingDataPointsFilters:
  - RandomSamplingDataPointsFilter:
      prob: 1

referenceDataPointsFilters:
  - MaxDistDataPointsFilter:
      maxDist: inf
  - RandomSamplingDataPointsFilter:
      prob: 1

matcher:
  KDTreeMatcher:
    knn: 3
    maxDist: 10.0
    epsilon: 3.33


outlierFilters:
  - TrimmedDistOutlierFilter:
     ratio: 0.9
  - SurfaceNormalOutlierFilter:
     maxAngle: 3.14

errorMinimizer:
  PointToPointErrorMinimizer

transformationCheckers:
  - DifferentialTransformationChecker:
      minDiffRotErr: 0.01
      minDiffTransErr: 0.1
      smoothLength: 10
  - CounterTransformationChecker:
      maxIterationCount: 500
  - BoundTransformationChecker:
      maxRotationNorm: 2
      maxTranslationNorm: 30

inspector:
#  VTKFileInspector
  NullInspector

logger:
#  FileLogger
  NullLogger
